modules/services/featureflag/src/main/java/au/com/shiftyjelly/pocketcasts/featureflag/providers/StoreFeatureFlagProvider.kt

```diff
@@ -0,0 +1,27 @@
+package au.com.shiftyjelly.pocketcasts.featureflag.providers
+
+import au.com.shiftyjelly.pocketcasts.featureflag.Feature
+import au.com.shiftyjelly.pocketcasts.featureflag.FeatureFlag
+import au.com.shiftyjelly.pocketcasts.featureflag.FeatureFlagProvider
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Used to set feature flag values in release builds.
+ * Any changes to the feature flags through FeatureFlagManager are not applied to feature flags
+ * provided through this provider.
+ */
+@Singleton
+class StoreFeatureFlagProvider @Inject constructor() : FeatureFlagProvider {
```

Comment:
I was tempted to name it `ReleaseFeatureFlagProvider` but since it might not be the only provider for a release build I chose a different name. Let me know if you have a better suggestion.